massdriver.yaml

```
@@ -42,20 +41,37 @@ params:
       type: string
       title: Tier
       description: Availability tier.
-      default: single-instance
+      default: BASIC
       $md.immutable: true
       oneOf:
-      - title: Single Instance
-        const: single-instance
-      - title: Highly Available
-        const: standard-ha
-    read_replicas:
-      type: integer
-      title: Read Replicas
-      description: Number of read replicas to provision. Must be an integer between 1 and 5.
-      minimum: 1
-      maximum: 5
-      default: 1
+        - title: Single Instance
+          const: BASIC
+        - title: Highly Available
+          const: STANDARD_HA
+  dependencies:
+    tier:
+      oneOf:
+        - properties:
+            tier:
+              title: Tier
+              oneOf:
+                - title: Highly Available
+                  const: STANDARD_HA
```

Did you test this in RJSF? I think the proper formatting would be this:
```suggestion
            tier:
              const: STANDARD_HA
```

---

I did yes. It looks like these do the same thing, your way is cleaner. 👍